classes/mix.bbclass

```diff
@@ -33,18 +35,13 @@ FILES_${PN}-dbg += "${APP_PREFIX}/${APPNAME}/${APPVERSION}/erts*/bin/.debug"
 FILES_${PN}-staticdev += "${APP_PREFIX}/${APPNAME}/${APPVERSION}/erts*/lib/lib*.a"
 FILES_${PN}-staticdev += "${APP_PREFIX}/${APPNAME}/${APPVERSION}/erts*/lib/internal/lib*.a"
 
-APP_HOME ?= "/run/${APPNAME}"
-MUTABLE_DIR ?= "/run/${APPNAME}"
-USER_CONFIG_FILE ?= "${sysconfdir}/${APPNAME}.conf"
-CONFFILES_${PN} += "${SYSCONFIG_PREFIX}/${APPNAME}.conf"
```

Comment:
Why did you remove this ? I think this should stay in the mix class so that a new version of a package build with mix does not overwrite the old configuration file.